packages/ckeditor5-list/src/documentlistproperties/documentliststylecommand.js

```diff
@@ -0,0 +1,141 @@
+/**
+ * @license Copyright (c) 2003-2022, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module list/documentlistproperties/documentliststylecommand
+ */
+
+import { Command } from 'ckeditor5/src/core';
+import { expandListBlocksToCompleteItems, getListItems } from '../documentlist/utils/model';
+import { getListTypeFromListStyleType } from './utils/style';
+
+/**
+ * The list style command. It changes `listStyle` attribute of the selected list items.
+ * It is used by the {@link module:list/documentlistproperties~DocumentListProperties list properties feature}.
+ *
+ * @extends module:core/command~Command
+ */
+export default class DocumentListStyleCommand extends Command {
+	/**
+	 * Creates an instance of the command.
+	 *
+	 * @param {module:core/editor/editor~Editor} editor The editor instance.
+	 * @param {String} defaultType The list type that will be used by default if the value was not specified during
+	 * the command execution.
+	 */
+	constructor( editor, defaultType ) {
+		super( editor );
+
+		/**
+		 * The default type of the list style.
+		 *
+		 * @protected
+		 * @member {String}
+		 */
+		this._defaultType = defaultType;
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	refresh() {
+		this.value = this._getValue();
+		this.isEnabled = this._checkEnabled();
+	}
+
+	/**
+	 * Executes the command.
+	 *
+	 * @param {Object} options
+	 * @param {String|null} [options.type] The type of the list style, e.g. `'disc'` or `'square'`. If `null` is specified, the default
```

Comment:
It would break the UI. And if UI is updated, it wouldn't work with `ListStyleCommand`. And if `ListStyleCommand` is updated, that would be breaking change.